src/panels/lovelace/cards/hui-entity-button-card.js

```diff
@@ -1,161 +0,0 @@
-import { html } from "@polymer/polymer/lib/utils/html-tag.js";
```

Comment:
When you rename files, use `git mv <old> <new>`, that way the history of the old file is transferred to the new one.